influxdb/influxdb.go

```diff
@@ -0,0 +1,217 @@
+package influxdb
+
+import (
+	"github.com/prismatik/jabba"
+)
+
+func Go() {
+	jabba.RunOrDie("wget", "--continue", "http://s3.amazonaws.com/influxdb/influxdb_latest_amd64.deb", "&&", "sudo", "dpkg", "-i", "influxdb_latest_amd64.deb")
```

Comment:
Why fetch the deb rather than using the repository? This way we don't get updates. They maintain an Ubuntu apt repo.

https://docs.influxdata.com/influxdb/v0.13/introduction/installation/